app/tests/unit/components/Form/ProjectContactForm.test.tsx

```diff
@@ -465,30 +465,26 @@ describe("The ProjectContactForm", () => {
       componentTestingHelper.environment.mock.getAllOperations()
     ).toHaveLength(3);
 
-    const firstMutationUnderTest =
-      componentTestingHelper.environment.mock.getAllOperations()[1];
-
-    expect(firstMutationUnderTest.fragment.node.name).toBe(
-      "deleteFormChangeWithConnectionMutation"
-    );
-
-    expect(firstMutationUnderTest.request.variables).toMatchObject({
-      input: {
-        id: "Form ID 1",
-      },
-    });
-
-    const secondMutationUnderTest =
-      componentTestingHelper.environment.mock.getAllOperations()[2];
-
-    expect(secondMutationUnderTest.fragment.node.name).toBe(
-      "deleteFormChangeWithConnectionMutation"
+    componentTestingHelper.expectMutationToBeCalled(
+      "deleteFormChangeWithConnectionMutation",
+      ...[
+        {
+          connections: [
+            'client:Test Project Revision ID:__connection_projectContactFormChanges_connection(formDataTableName:"project_contact")',
+          ],
+          input: {
+            id: "Form ID 1",
+          },
+        },
+        {
+          connections: [
+            'client:Test Project Revision ID:__connection_projectContactFormChanges_connection(formDataTableName:"project_contact")',
+          ],
+          input: {
+            id: "Form ID 2",
+          },
+        },
+      ]
     );
```

Comment:
We should be testing the 2 mutations separately, I think giving an array in the `variables` parameter of the helper confuses it

```suggestion
    componentTestingHelper.expectMutationToBeCalled(
      "deleteFormChangeWithConnectionMutation",
        {
          connections: expect.any(Array),
          input: {
            id: "Form ID 1",
          },
        }
    );
    
   componentTestingHelper.expectMutationToBeCalled(
      "deleteFormChangeWithConnectionMutation",
        {
          connections: expect.any(Array),
          input: {
            id: "Form ID 2",
          },
        }
    );
```